blobit-core/src/main/java/org/blobit/core/api/NamedObjectMetadataFilter.java

```diff
@@ -0,0 +1,28 @@
+/*
+ Licensed to Diennea S.r.l. under one
+ or more contributor license agreements. See the NOTICE file
+ distributed with this work for additional information
+ regarding copyright ownership. Diennea S.r.l. licenses this file
+ to you under the Apache License, Version 2.0 (the
+ "License"); you may not use this file except in compliance
+ with the License.  You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing,
+ software distributed under the License is distributed on an
+ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ KIND, either express or implied.  See the License for the
+ specific language governing permissions and limitations
+ under the License.
+
+ */
+package org.blobit.core.api;
+
+/**
+ * A Filter over ObjectMetadata
+ * @author eolivelli
+ */
+public interface NamedObjectMetadataFilter {
```

Comment:
Is it used anywere? Filter with no method?

Reply:
It is a left over, in this patch I wanted to introduce an API to query for named blobs but I have dropped it in order to focus on the new schema and on *append* API.

I will drop